be/src/olap/rowset/segment_v2/segment.cpp

```diff
@@ -367,14 +367,14 @@ Status Segment::lookup_row_key(const Slice& key, bool with_seq_col, RowLocation*
 
     DCHECK(_pk_index_reader != nullptr);
     if (!_pk_index_reader->check_present(key_without_seq)) {
-        return Status::NotFound("Can't find key in the segment");
+        return Status::Error<ErrorCode::KEY_NOT_FOUND>("Can't find key in the segment");
```

Comment:
Use `ERROR_CTOR(KeyNotFound, KEY_NOT_FOUND)` is more clear?

Reply:
It seems that `ERROR_CTOR ` is for Errors defined in TStatus. Here, `KEY_NOT_FOUND` and `KEY_ALREADY_EXSITS` are be internal errors

Reply:
got it